mxd/modules/connector/azurite-container.tf

```diff
@@ -0,0 +1,75 @@
+#
+#  Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+resource "kubernetes_job" "azurite-container" {
+
+  metadata {
```

Comment:
here, i would add an explicit `dependsOn` entry, to make sure this job runs after the azurite container was deployed

Reply:
Here `module.azurite` is not visible. Added explicit dependency in connector. So now, connectors depends on azurite module.